Controllers/Backend/SwagBackendOrder.php

```diff
@@ -597,6 +598,12 @@ private function createBasketCalculationResult(
             foreach ($tax as $price) {
                 $totalPriceResult->addTax($price->getTaxRate(), $price->getTax());
             }
+
+            $shippingCostsNet = 0;
+            foreach ($tax as $shippingProportional) {
+                $shippingCostsNet += $shippingProportional->getNetPrice();
+            }
```

Comment:
I use this logic to get the correct calculated shipping-netto-cost which is stored in the database: https://github.com/shopware5/shopware/blob/5.7/engine/Shopware/Controllers/Frontend/Checkout.php#L1231-L1235

Reply:
removed because logic didn't work 100%